Description: Remove the dependency on the Activation Framework (integrated to Java 6 and later)
Author: Emmanuel Bourg
--- a/build.xml
+++ b/build.xml
@@ -118,7 +118,6 @@
   <path id="compile.classpath">
     <pathelement location="${build.home}/classes"/>
     <pathelement location="${javamail.jar}"/>
-    <pathelement location="${activation.jar}"/>
     <pathelement location="${jaxp.jaxp.jar}"/>
     <pathelement location="${jms.jar}"/>
     <pathelement location="${jmx.jar}"/>
--- a/tests/build.xml
+++ b/tests/build.xml
@@ -35,7 +35,6 @@
   <property name="checkstyle.version" value="4.1"/>
   <property name="checkstyle.jar" location="${m2_repo}/checkstyle/checkstyle/${checkstyle.version}/checkstyle-${checkstyle.version}.jar"/>
   <property name="javamail.jar" location="${m2_repo}/javax/mail/mail/1.4.3/mail-1.4.3.jar"/>
-  <property name="activation.jar" location="${m2_repo}/javax/activation/activation/1.1/activation-1.1.jar"/>
   <property name="junit.version" value="3.8.1"/>
   <property name="junit.jar" location="${m2_repo}/junit/junit/${junit.version}/junit-${junit.version}.jar"/>
 
@@ -70,7 +69,6 @@
     <pathelement location="${clover.jar}"/>
     <pathelement location="${junit.jar}"/>
     <pathelement location="${javamail.jar}"/>
-    <pathelement location="${activation.jar}"/>
     <pathelement path="${java.class.path}"/>
   </path>
 
